test/extended/builds/service.go

```diff
@@ -43,14 +42,20 @@ RUN curl -vvv hello-openshift:6379
 
 		g.Describe("with a build being created from new-build", func() {
 			g.It("should be able to run a build that references a cluster service", func() {
-				g.By("standing up a new hello world service")
-				err := oc.Run("new-app").Args("--name", "hello-openshift", k8simage.GetE2EImage(k8simage.Redis)).Execute()
+				g.By("standing up a new hello world redis service")
```

Comment:
I don't understand why we are adding the `redis` string here, when the rest of this PR seems to be removing Redis from the test-case.

Reply:
an oversight on my part...